services/ops/LogStatisticsAgent/README.md

```diff
@@ -0,0 +1,38 @@
+## Log Statistics Agent
+
+The Log Statistics agent periodically reads the "volttron.log" file, computes the size delta from the previous hour 
```

Comment:
is "periodically" hourly? If it is configurable or not hourly, are the comparisons always hourly? What are the units of the delta (lines? bytes?)

Is that blank you looking for feedback? My best guess would be something like:

> This agent can be useful for detecting unexpected changes to the system which may be an indication of some sort of failure or breach.